src/Scripts/Game/Components/Vehicle/EL_LicensePlateManagerComponent.c

```diff
@@ -0,0 +1,122 @@
+[EntityEditorProps(category: "EL/LicnesePlateManagerComponent", description: "This renders license plates on your vehicle.", color: "0 0 255 255")]
+class EL_LicensePlateManagerComponentClass: ScriptComponentClass
+{
+};
+
+[BaseContainerProps()]
+class EL_LicensePlateComponent
+{
+	[Attribute(uiwidget: UIWidgets.Auto)]
+	ref EL_PointInfo m_Point;
+	
+	EL_LicensePlateEntity m_Object;
+	
+	void ~EL_LicensePlateComponent()
+	{
+		if (m_Object)
+		{
+			delete m_Object;
+		}
+	}
+}
+
+class EL_PointInfo: PointInfo
+{
+}
+
+class EL_LicensePlateManagerComponent: ScriptComponent
+{
+	[Attribute(uiwidget: UIWidgets.Auto)]
+	ref array<ref EL_LicensePlateComponent> m_Plates;
+	
+	[Attribute("{E95486C43308F36B}Prefabs/Vehicles/LicensePlate/LicensePlate.et")]
+	protected ResourceName m_LicensePlatePrefab;
+	
+	[RplProp(onRplName: "OnRegistrationUpdated")]
+	string m_Registration;
+	
+	override void EOnInit(IEntity owner)
+	{
+		Resource resource = Resource.Load(m_LicensePlatePrefab);
+		
+		if (!resource.IsValid()) return;
+		
+		EntitySpawnParams params();
+		params.Parent = owner;
+		params.TransformMode = ETransformMode.LOCAL;
+		
+		RplComponent rpl = RplComponent.Cast(owner.FindComponent(RplComponent));
+		
+		if (rpl && rpl.IsMaster())
```

Comment:
This should be swapped so the RplComponent is checked only when the game is playing and not inside the World Editor window in Edit Mode.